### PR TITLE
Domain change in WireXNetworkForensicPlatform and MicrosoftDefenderforCloudApps

### PR DESCRIPTION
**Pull request by v-prasadboke**
Testing Doc Link for WireX Network Forensic Platform:- https://microsoft-my.sharepoint.com/:w:/p/v-prasadboke/Ef9UQsvFpOFOohM3O6V55mUBzvhAfiTXf3M1DT5veR0J4A?e=q4VORO

Testing Doc Link for Microsoft Defender for Cloud Apps :- https://microsoft-my.sharepoint.com/:w:/p/v-prasadboke/ETsumwWMcJtPkjnFFbDzZvYBZKLhh2wgW2SaJf0vE6RjXg?e=pjhYEx

The changes are done to correct the domain categories for WireX Network Forensic Platform and Microsoft Defender for Cloud Apps solutions.

### CODE REVIEW

File: Solutions/Microsoft Defender for Cloud Apps/Package/createUiDefinition.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs the data connector for ingesting Microsoft Defender for Cloud Apps logs into Microsoft Sentinel, using Codeless Connector Platform and Native Sentinel Polling. After installing the solution, configure and enable this data connector by following guidance in Manage solution view."
+              "text": "This Solution installs the data connector for Microsoft Defender for Cloud Apps. You can get Microsoft Defender for Cloud Apps custom log data in your Microsoft Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) SecurityAlert (MCAS) McasShadowItReporting in your Microsoft Sentinel / Azure Log Analytics workspace."
```
Comment (v-sabiraj) [inline Solutions/Microsoft Defender for Cloud Apps/Package/createUiDefinition.json:63]: @v-prasadboke please revert the description changes,thanks.

File: Solutions/Microsoft Defender for Cloud Apps/Package/createUiDefinition.json [hunk 2]
```diff
@@ -88,7 +88,7 @@
             "name": "workbooks-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This solution installs workbook(s) to help you gain insights into the telemetry collected in Microsoft Sentinel. After installing the solution, start using the workbook in Manage solution view."
+              "text": "This Microsoft Sentinel Solution installs workbooks. Workbooks provide a flexible canvas for data monitoring, analysis, and the creation of rich visual reports within the Azure portal. They allow you to tap into one or many data sources from Microsoft Sentinel and combine them into unified interactive experiences."
```
Comment (v-sabiraj) [inline Solutions/Microsoft Defender for Cloud Apps/Package/createUiDefinition.json:91]: @v-prasadboke, please revert this description as well.

**Comment by v-dvedak** [comment]
Changes for Microsoft Defender for Cloud Apps looks good

**Comment by v-dvedak** [comment]
@v-prasadboke, I see Azure Sentinel being used in solution "WireX Network Forensics Platform". Please correct it to Microsoft Sentinel.

**Comment by v-prasadboke** [comment]
@v-dvedak done changing the Azure sentinel to Microsoft sentinel and even updated the testing doc for Microsoft Defender for Cloud Apps

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**